public-ui/tslint.json

```diff
@@ -0,0 +1,153 @@
+{
```

Comment:
This is the lint file I was thinking it may be possible to share.

Reply:
shared.